src/components/AssetManagementPanel/AssetManagementForm/FormVariants/EditableAssetTitle/EditableAssetTitle.tsx

```diff
@@ -23,16 +31,21 @@ export const EditableAssetTitle = ({ role, value, newValue, isEditable, onSetNew
   return (
     <AssetTitle role={role} address={value}>
       <div className="row no-gutters align-items-center">
-        <div className="col">
+        <InputEditableWrapper className="col">
           <InputEditableAssetTitle
             data-testid={`editable-input-${role.toLowerCase()}`}
             value={newValue}
+            errorState={errorState === "ERROR"}
+            placeholder={`Input ${role}'s address`}
             onChange={(event) => {
               if (!onSetNewValue) return;
               onSetNewValue(event.target.value);
             }}
           />
-        </div>
+          {errorState === "ERROR" && (
```

Comment:
Think that boon has a workflow for modal already, please check with @simboonlong

Reply:
ok got it!